jekyll/_cci2/arm_resources.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: classic-docs
+title: "ARM resources"
+short-title: "Using ARM resources on CircleCI"
+description: "Using ARM resources on CircleCI"
+version:
+- Cloud
+---
+
+# Overview
+
+This document will walk you through the setup steps required to use an ARM
+resource on CircleCI. Current, ARM resources are not available on CircleCI
+Server.
```

Comment:
Let’s say “Arm resources are not available on CircleCI Server 1.x or 2.x. If you are interested in using Arm with CircleCI Server 3.0, please contact your CircleCI Customer Success Manager or [fill out this contact form](https://circleci.com/contact-us/).